cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -10,6 +10,11 @@ describe('FluxQueryBuilder', () => {
           cy.setFeatureFlags({
             newDataExplorer: true,
           }).then(() => {
+            const writeData = []
+            for (let i = 0; i < 30; i++) {
+              writeData.push(`ndbc,air_temp_degc=${i}_degrees station_id=${i}`)
```

Comment:
```suggestion
              writeData.push(`ndbc,station_id=${i} air_temp_degc=${i}_degrees`)
```

Or change the line protocol to keep `air_temp_degc` as field and `station_id` as tag key